lib/profile.js

```diff
@@ -0,0 +1,8 @@
+const apiRequest = require('./apiRequest');
+
+async function getProfile(account_name) {
```

Comment:
this could go to `lib/user.js` instead of having a `profile.js`

Reply:
noted